src/main/java/seedu/address/logic/commands/CommandResult.java

```diff
@@ -4,38 +4,83 @@
 
 import java.util.Objects;
 
+import seedu.address.model.client.Client;
+
 /**
  * Represents the result of a command execution.
  */
 public class CommandResult {
 
     private final String feedbackToUser;
 
-    /** Help information should be shown to the user. */
+    private final Client targetClient;
+
+    /**
+     * Selects a particular client
+     */
+    private final boolean select;
```

Comment:
Do you think we should rename the boolean to isSelected, to stick to coding standards?

Reply:
Ooh, great idea!

Reply:
`select` is not used in the conventional sense here, it follows the same convention as the other two boolean state variables `showHelp` and `exit`. 

It is supposed to represent the state of the command being entered.